media/js/firefox/mobile-download-desktop.js

```diff
@@ -0,0 +1,34 @@
+(function($) {
+    'use strict';
+
+    var appStoreURLs = {
+        'gb': 'https://app.adjust.com/2uo1qc?campaign=Heartbeat&adgroup=iOS_App_Store&creative=badge_de&fallback=https%3A%2F%2Fitunes.apple.com%2Fgb%2Fapp%2Fapple-store%2Fid989804926%3Fpt%3D373246%26ct%3Dadjust_tracker%26mt%3D8',
```

Comment:
Should this be `&creative=badge_gb`? Seems there are lots of occurrences of `badge_de` below too.